cfpb/consumerfinance.gov
# Prepaid agreements: sort issuer names in filter

A stakeholder noticed that the values are no longer showing in alphabetical order in the issuer name filter on the prepaid agreements database page. I think the ordering may have originally occurred in the Django view and was affected by unrelated changes to that code. This PR moves the alphabetization to the front end, adding a sort filter when the multiselect is generated in the Jinja template.

---

## Additions

- Sort filter for issuer names in Jinja2 template


## How to test this PR

1. Check out branch
2. Navigate to [prepaid agreements database](http://localhost:8000/data-research/prepaid-accounts/search-agreements/) and verify that values appear in alphabetical order in issuer name multiselect



## Screenshots

### Unsorted issuer names on production
<img width="369" alt="Screen Shot 2021-11-18 at 12 02 15 PM" src="https://user-images.githubusercontent.com/778171/142488483-83ee83e1-1d6e-43c1-a189-41af81f14a86.png">



### Sorted issuer names in this branch

<img width="404" alt="Screen Shot 2021-11-18 at 12 01 44 PM" src="https://user-images.githubusercontent.com/778171/142488495-07955bb1-8653-4aa9-815a-d4d418c08d1e.png">
